hadoop-hdds/interface-client/src/main/proto/DatanodeClientProtocol.proto

```diff
@@ -391,7 +393,7 @@ enum ChecksumType {
 
 message  WriteChunkRequestProto  {
   required DatanodeBlockID blockID = 1;
-  required ChunkInfo chunkData = 2;
+  optional ChunkInfo chunkData = 2;
```

Comment:
isn't this backword incompatible?

Reply:
Changed this from required to optional is compatible. The previous interface WriteChunkRequestProto must have chunkData. Init stream will use WriteChunkRequestProto, but do not require chunkData. Just like we implemented in the [POC code](https://github.com/captainzmc/hadoop-ozone/blob/streaming-poc-2/hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java#L162).